Modules/IO/MeshBase/include/itkMeshIOTestHelper.h

```diff
@@ -375,59 +375,59 @@ TestBaseClassMethodsMeshIO(typename TMeshIO::Pointer meshIO)
   return EXIT_SUCCESS;
 }
 
-void
-AllocateBuffer(void ** data, itk::IOComponentEnum componentType, itk::SizeValueType bufferSize)
+namespace itk
+{
+namespace MeshIOTestHelper
```

Comment:
Is there some reason why this method should be included in an additional namespace?

Not sure why I did not include in the `itk` namespace `TestBaseClassMethodsMeshIO` as well.

Reply:
> Is there some reason why this method should be included in an additional namespace?

Thanks for asking @jhlegarreta 

I did not want to keep `AllocateBuffer` in the global namespace, because I think it may easily cause naming conflicts with third party libraries (now or in the future). However, I did not want to place it _directly_ in the _root_ of the `itk` namespace (as in "itk::AllocateBuffer"), because it is not part of the public ITK interface. And "itk::AllocateBuffer" might also conflict with other parts of ITK  (now or in the future).  So to avoid any conflict, I chose `itk::MeshIOTestHelper::AllocateBuffer`. This very much clarifies where the function comes from. And it prevents any naming conflict. Is that OK to you?

Reply:
@N-Dekker thanks for the explanation. Makes sense.